docs/src/content/docs/it/guides/pages.mdx

```diff
@@ -104,15 +104,37 @@ Le seguenti proprietà differiscono dal frontmatter di Markdown:
 
 - La proprietà [`slug`](/it/reference/frontmatter/#slug) non è supportata e viene impostata automaticamente in base all'URL della pagina personalizzata.
 - L'opzione [`editUrl`](/it/reference/frontmatter/#editurl) richiede un URL per visualizzare un link di modifica.
-- La proprietà [`sidebar`](/it/reference/frontmatter/#sidebar) non è supportata. Nel frontmatter di Markdown, questa opzione consente la personalizzazione dei [gruppi di link generati automaticamente](/it/reference/configuration/#sidebar), che non è applicabile alle pagine che utilizzano il componente `<StarlightPage />`.
-
-{/* ##### `sidebar` */}
-
-{/* **tipo:** `SidebarEntry[] | undefined` */}
-{/* **predefinito:** la barra laterale generata in base alla [configurazione globale `sidebar`](/reference/configuration/#sidebar) */}
-
-{/* Fornisce una barra laterale di navigazione personalizzata per questo sito. */}
-{/* Se non impostato, la pagina utilizzerà la barra laterale globale predefinita. */}
+- La proprietà del frontmatter [`sidebar`](/it/reference/frontmatter/#sidebar) per personalizzare come appare la pagina in [gruppi di link autogenerati](/it/reference/configuration/#sidebar) non è disponibile. Le pagine che utilizzano il componente `<StarlightPage />` non fanno parte di una collezione e non possono essere aggiunte a un gruppo di barra laterale autogenerato.
+
+##### `sidebar`
+
+**tipo:** `SidebarEntry[]`  
+**predefinito:** la barra laterale generata in base alla [configurazione globale `sidebar`](/it/reference/configuration/#sidebar)
+
+Fornisce una barra laterale di navigazione personalizzata per questa pagina.
+Se non impostato, la pagina utilizzerà la barra laterale globale predefinita.
+
+Ad esempio, la pagina seguente sovrascrive la barra laterale predefinita con un link alla homepage e un gruppo di link a diverse costellazioni.
+La pagina corrente nella barra laterale è impostata utilizzando la proprietà `isCurrent` e un `badge` opzionale è stato aggiunto a un elemento di link.
+
+```astro {3-13}
+<StarlightPage
+	frontmatter={{ title: 'Orione' }}
+	sidebar={[
+		{ label: 'Home', href: '/' },
+		{
+			label: 'Costellazioni',
+			items: [
+				{ label: 'Andromeda', href: '/andromeda/' },
+				{ label: 'Orione', href: '/orion/', isCurrent: true },
+				{ label: 'Orsa Minore', href: '/ursa-minor/', badge: 'Stub' },
```

Comment:
I guess we should also translate the badge ([source for this translation](https://it.wikipedia.org/wiki/Aiuto:Abbozzo)):

```suggestion
				{ label: 'Orsa Minore', href: '/ursa-minor/', badge: 'Abbozzo' },
```

Reply:
totally right :)